benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+                        n_rows, n_cols
+                    ),
+                    f"dec_cross_rle_opr_str_{n_cols}_{n_rows}": gen_dec_cross_rle_opr_str(
+                        n_rows, n_cols
+                    ),
+                }
+            )
+
+    return result
+
+
+def write_to_csv(data, file_path):
```

Ruff (0.11.9)

Missing return type annotation for public function `write_to_csv`

Add return type annotation: `None`

(ANN201)